lib/features/build_server/compiler.dart

```diff
@@ -207,3 +235,24 @@ class _ProcessInvocation {
     return '$executable ${args.join(' ')}';
   }
 }
+
+/// The status delegate of the `Compiler`.
+mixin CompilerStatusDelegate {
```

Comment:
Are you sure this is the correct file for this?

Reply:
Well, yes :) We do you think it should be moved somewhere else? It belongs strictly to `Compiler` and as far as I'm not mistaken Dart encourages to put logically related parts of your code in the same file, doesn't it?

Reply:
It can be easily overlooked, thats why I’m asking. You may resolve this